starfish/pipeline/filter/gaussian_low_pass.py

```diff
@@ -0,0 +1,49 @@
+from skimage.filters import gaussian
+import numpy
+
+from starfish.munge import swap
+from ._base import FilterAlgorithmBase
+
+
+# todo how do we make these methods available in the API?
+# the starfish import is a bit kludgy because it means we need to:
+# starfish.GaussianFilter.filter()
+class GaussianFilter(FilterAlgorithmBase):
```

Comment:
This is called GaussianFilter, but only does a lowpass filter. I see two options, GaussianFilter does either high-pass or low-pass, and takes in an argument like filter_type. OR, we rename this to GaussianLowPassFilter. We make a new class called GaussianHighPassFilter. I think I prefer the former -- what do you think?